ros2topic/ros2topic/verb/find.py

```diff
@@ -33,6 +35,7 @@ def add_arguments(self, parser, cli_name):
         parser.add_argument(
             '--include-hidden-topics', action='store_true',
             help='Consider hidden topics as well')
+        add_strategy_node_arguments(parser)
```

Comment:
This change doesn't seem to be related to the topic of the PR. Please consider separating it.